tensorflow/compiler/jit/mark_for_compilation_pass.cc

```diff
@@ -1714,7 +1761,300 @@ Status MarkForCompilationPass::RunForTest(
   return MarkForCompilation(options, debug_options);
 }
 
+absl::flat_hash_map<string, std::vector<string>>* CreateWhitelist() {
+  // Table format: category name: {list of TF operations in that category}
+  absl::flat_hash_map<string, std::vector<string>>* result =
```

Comment:
Should this be static? At the moment this seems to leak on every invocation.

Reply:
I modified it. See other comment: https://github.com/tensorflow/tensorflow/pull/34655#discussion_r353833354